lib/libdl/include/dlfcn.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2019 Linaro limited
+ */
+
+#ifndef _DLFCN_H_
+#define _DLFCN_H_
+
+/* Relocations are performed when the object is loaded. */
+#define RTLD_NOW	2
+/* All symbols are available for relocation processing of other modules. */
+#define RTLD_GLOBAL	0x100
+/* Do not unload the shared object during dlclose(). */
+#define RTLD_NODELETE	0x1000
+/* Other flags are not supported */
+
+void *dlopen(const char *filename, int flags);
```

Comment:
It could be worth mentioning that currently flags has to be `RTLD_NOW | RTLD_GLOBAL | RTLD_NODELETE`.

Reply:
Already documented in `lib/libutee/include/pta_system.h`, but here is probably a better place (visible to the user of the API). I'll add the comment here. Should I drop the same in `pta_system.h`?